configs/deprecated/example/se.py

```diff
@@ -142,27 +140,27 @@ def get_processes(args):
         print("number of benchmarks not equal to set num_cpus!")
         sys.exit(1)
 
-    for app in apps:
-        try:
-            if get_runtime_isa() == ISA.ARM:
-                exec(
-                    "workload = %s('arm_%s', 'linux', '%s')"
-                    % (app, args.arm_iset, args.spec_input)
-                )
-            else:
-                # TARGET_ISA has been removed, but this is missing a ], so it
-                # has incorrect syntax and wasn't being used anyway.
-                exec(
-                    "workload = %s(buildEnv['TARGET_ISA', 'linux', '%s')"
-                    % (app, args.spec_input)
-                )
-            multiprocesses.append(workload.makeProcess())
-        except:
-            print(
-                f"Unable to find workload for {get_runtime_isa().name()}: {app}",
-                file=sys.stderr,
-            )
-            sys.exit(1)
+    # for app in apps:
+    #    try:
+    #        if get_isa_from_str(args.isa) == ISA.ARM:
+    #            exec(
+    #                "workload = %s('arm_%s', 'linux', '%s')"
+    #                % (app, args.arm_iset, args.spec_input)
+    #            )
+    #        else:
+    #            # TARGET_ISA has been removed, but this is missing a ], so it
+    #            # has incorrect syntax and wasn't being used anyway.
+    #            exec(
+    #                "workload = %s(buildEnv['TARGET_ISA', 'linux', '%s')"
+    #                % (app, args.spec_input)
+    #            )
+    #        multiprocesses.append(workload.makeProcess())
+    #    except:
+    #        print(
+    ##            f"Unable to find workload for {args.isa}: {app}",
+    #           file=sys.stderr,
+    #        )
+    #        sys.exit(1)
```

Comment:
I really don't know what this code does but it behaves differently depending on whether your targeting the arm ISA or nto